client/src/components/Feed/Post.js

```diff
@@ -604,6 +608,17 @@ const Post = ({
             )}
             <div className="pre-header post-page">
               <span>{t(`feed.${objective}`)}&nbsp;&nbsp;•</span>
+              <span>
+                &nbsp;&nbsp;
```

Comment:
try <Space/> or other antd components? `&nbsp;` might not ideal

Reply:
tried using WhiteSpace - puts it into next line, antd Space doesn't help

Reply:
the [antd <Space>](https://ant.design/components/space/) might be useful here